guides_and_processes/engineering_meetings.md

```diff
@@ -1,3 +1,240 @@
+
+**Who Should Attend:**  
+- **Mandatory:** Architects, Engineering Manager, Backend & Frontend Developers, Product Owner  
+- **Recommended:** Designers, Design Manager (if design tasks are included), Testers/QA (if testing tasks are included)
+  
+**Inputs / Preparation:**  
+- Reviewed and prioritized backlog items.
+- Acceptance criteria and estimates in place. (can be completed by engineers separate from meetings)
+- Any known dependencies identified.
+
+**Expected Outputs:**  
+- A sprint backlog with clearly defined tasks ready to be worked on in the next spring.
+- A shared understanding of the sprint goals and commitments.
+
+**Frequency:**  
+- Start of each sprint (e.g., every 2 weeks).
+
+---
+
+### Backlog Grooming
+
+**Purpose:**  
+- Review candidate tickets/epics that have been prepared to ensure they have clear acceptance criteria, defined scope, and no outstanding dependencies.  
+- Discuss estimates and confirm if the size of each task is manageable within a single sprint.  
+- Validate that each item aligns with overall product and architectural goals.  
+- Prioritize or re-prioritize items based on current project needs and constraints.  
+- Identify any items needing further refinement before they can be moved into a “Ready for Dev” state.
+
+**What to Do:**  
+- Review candidate tickets/epics, clarify acceptance criteria, confirm tasks have been estimated by the engineering team.
+- Ensure upcoming work is well-defined and aligns with product goals and architectural standards.
+- Ensure each task/epic has the minimum required criteria as defined by the team (see separate document)
+
+**What Not to Do:**  
+- Do not involve the entire dev team—this is a focused session for shaping work.
+- Avoid unrelated technical deep-dives not pertinent to prioritizing and refining tickets.
+
+**Who Should Attend:**  
+- **Mandatory:** Architects, Engineering Manager, Product Owner  
+- **Optional:** Others may be invited for specific expertise if needed.
+
+**Inputs / Preparation:**  
+- Product Owner prepares candidate backlog items.
+- Architects/Eng. management review items for feasibility and clarity before the meeting.
+
+**Expected Outputs:**  
+- A prioritized and well-defined backlog of items suitable for the next sprint planning, this becomes the runway for engineering and does not need to be limited to just the next sprint, attempt to define a well groomed backlog for up to 3 sprints ahead, any further planning could result in wasted effort if the product direction changes. The length of the engineering runway will be determined by the engineering teams velocity over time.
+- Clarity on scope, acceptance criteria, and readiness of items.
+
+**Frequency:**  
+- Weekly or Bi-weekly as needed before Sprint Planning, this does not need to align with the sprint cycle. 
+
+---
+
+### Tech Discussion
+
+**Purpose:**  
+- Discuss and align on technical challenges, architectural decisions, and implementation approaches.
```

Comment:
Also open for other technical topic

- Share and discuss interesting or useful topics that can benefit the team, such as new tools, techniques, or industry best practices.